functions/processQueueItem.js

```diff
@@ -91,12 +136,6 @@ module.exports.handler = async function(
     });
     const timingSubmitted = Date.now() - timingSubmittedStart;
 
-    const { QueueUrl } = await SQS.getQueueUrl({
-      QueueName: QUEUE_NAME
-    }).promise();
-
-    await SQS.deleteMessage({ QueueUrl, ReceiptHandle }).promise();
```

Comment:
We don't have to explicitly delete messages any more, here. The built-in poller automatically deletes them if this function completes without error.